lib/assets.rb

```diff
@@ -32,39 +35,76 @@ def html(script, **needs)
     context.eval(Snabberb.html_script(script, **needs))
   end
 
-  def build
-    return [@bundle_path] if @precompiled
+  def game_builds
```

Comment:
i think we might want to get rid of the source maps and instead make source maps for the uglifier